### DISCUSSION TITLE
3D var chart in superset?

### DISCUSSION THREAD

**Discussion by OCM1** [apache/superset#27208] [Q&A / Help]
I want to visualise data in a 3d bar chart as shown in the image. Is there any way to do that with a plug-in or something? Thanks!

![1](https://github.com/apache/superset/assets/160890182/2d4803e5-6cab-4c21-90b6-21fdeac3d086)

**Comment by sandeepr43** [comment]
+1 
any option for 3D charts like above would be helpful

**Comment by pm-at** [comment]
Superset is using ECharts and apparently ECharts already has a chart i.e. `bar3D` for this:
https://echarts.apache.org/examples/en/editor.html?c=bar3d-punch-card&gl=1

Are there any plans on integrating it? This type of chart is superuseful when we don't want to stack categories but the plotting categories on individual bars on x axis would make it too crowded.
